File: java/src/main/java/com/dsi/laboreos/model/MomentoLaboreo.java
package com.dsi.laboreos.model;

public class MomentoLaboreo {
    private String nombre;
    private String descripcion;

    public MomentoLaboreo(String nombre, String descripcion) {
        this.nombre = nombre;
        this.descripcion = descripcion;
    }

    public String getNombre() {
        return nombre;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public String mostrarMomentoLaboreo() {
        return nombre;
    }
}

File: python/laboreos/models/MomentoLaboreo.py
from django.db import models


class MomentoLaboreo(models.Model):
    nombre = models.CharField(max_length=100, primary_key=True)
    descripcion = models.TextField()
    
    class Meta:
        db_table = 'momentos_laboreo'
    
    def mostrar_momento_laboreo(self):
        return self.nombre

